snapcraft/internal/lifecycle/_status_cache.py

```diff
@@ -34,8 +35,8 @@ def __init__(self, config: _config.Config) -> None:
         """
         self.config = config
         self._steps_run = dict()  # type: Dict[str, Set[steps.Step]]
-        self._outdated_reports = collections.defaultdict(dict)  # type: _Report
-        self._dirty_reports = collections.defaultdict(dict)  # type: _Report
+        self._outdated_reports = collections.defaultdict(dict)  # type: _OutdatedReport  # noqa
```

Comment:
why noqa or why noqa with no qualifier, and if it is to prevent a line break, I'd rather change the line limit to 99 :wink:
Use your parens until then :laughing:

Reply:
Fixed!